media/js/src/CollectionTab.jsx

```diff
@@ -114,6 +114,7 @@ export default class CollectionTab extends React.Component {
                 <AssetFilter
                     assets={this.props.assets}
                     assetCount={this.props.assetCount}
+                    hidePagination={!!this.state.selectedAsset}
```

Comment:
I'm not familiar with this syntax...double negation?

Reply:
Yeah, this turns a nullable object into a boolean. If there is a selectedAsset, this will be `true`, otherwise it will be `false`.